package.json

```diff
@@ -13,37 +13,37 @@
   "author": "Algolia <support@algolia.com>",
   "license": "MIT",
   "repository": "algolia/instantsearch.js",
-  "main": "dist-es5-module/index.js",
-  "module": "./es/index.js",
-  "jsdelivr": "dist/instantsearch.min.js",
+  "main": "dist/cjs/index.js",
+  "module": "dist/es/index.js",
+  "jsdelivr": "dist/umd/instantsearch.production.min.js",
   "sideEffects": false,
   "scripts": {
-    "build": "./scripts/build.sh",
     "dev": "./scripts/dev.sh",
-    "dev-novel": "NODE_ENV=development webpack-dev-server --config dev/webpack.config.js",
-    "doctoc": "doctoc --no-title --maxlevel 3 README.md CONTRIBUTING.md",
+    "build": "yarn build:cjs && yarn build:es && yarn build:umd",
+    "build:umd": "BABEL_ENV=umd rollup --config scripts/rollup.config.js",
+    "build:cjs": "BABEL_ENV=cjs babel src --out-dir dist/cjs --ignore src/index.es.js,__tests__,__mocks__ --quiet",
+    "build:es": "BABEL_ENV=es babel src --out-dir dist/es --ignore src/index.es.js,__tests__,__mocks__ --quiet && babel src/index.es.js --out-file dist/es/index.js --quiet",
```

Comment:
The `BABEL_ENV` is not set for the second `babel` call. It means that the main entry is not ESM compliant. See [output](https://deploy-preview-3260--instantsearchjs.netlify.com/v2/lib/es/index.js).

Reply:
That's right, thank you!

Reply:
That's right, thank you!